awx/ui/client/lib/components/layout/_index.less

```diff
@@ -142,12 +142,19 @@
         overflow-x: hidden;
     }
 
+    &-content {
+        flex: 1;
+    }
+
     &-footer {
+        margin-top: 10px;
+        padding-bottom: 5px;
```

Comment:
20px here too, thanks!

Reply:
Also, could you add `padding-right: 20px` too? That way it will line up with the right hand side of the panel border. 

With this amount of padding it should look like this: 
![image](https://user-images.githubusercontent.com/7010629/30342254-106d7a88-97ae-11e7-808d-1ef348a15472.png)